AppPackage/Package.swift

```diff
@@ -0,0 +1,36 @@
+// swift-tools-version: 5.7
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "AppPackage",
+    platforms: [
+        .iOS(.v14),
+    ],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "AppPackage",
+            targets: ["AppPackage"]),
```

Comment:
<!--
  0 failure: 
  1 warning:  Multiline argumen...
  
  
  DangerID: danger-id-Danger;
  File: AppPackage/Package.swift;
  Line: 15;
-->

- :warning: Multiline arguments should have their surrounding brackets in a new line. (`multiline_arguments_brackets`)